### PR TITLE
OV Operator 0.2

### PR DESCRIPTION
**Pull request by dtrawins**

### CODE REVIEW

File: extras/openvino-operator-openshift/tests/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+```bash
+export REGISTRY=quay.io
+export IMG=$REGISTRY/ovms-operator:latest
+make docker-build docker-push IMG=$IMG
+```
+
+## Deploy in Kubernetes the operator without OLM
+```bash
+export IMG=quay.io/openvino/ovms-operator:0.1.0
```
Comment (mzegla) [inline extras/openvino-operator-openshift/tests/README.md:13]: Will this still be 0.1.0 ?

File: extras/openvino-operator-openshift/tests/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+- podman
+- [opm](https://github.com/operator-framework/operator-registry/releases/latest) 
+- registry with push permissions
+
+```bash
+export REGISTRY=quay.io
+export IMG_BUNDLE=$REGISTRY/ovms-operator-bundle:latest
+export IMG_CATALOG=$REGISTRY/test-catalog:latest
+git clone https://github.com/operator-framework/community-operators
+cd community-operators/upstream-community-operators/ovms-operator/0.1.0
```
Comment (mzegla) [inline extras/openvino-operator-openshift/tests/README.md:30]: Will this still be 0.1.0 ?

File: extras/openvino-operator-openshift/tests/sample1.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  resources:
+    limits:
+      cpu: 1
+      memory: 512Mi
+  rest_port: 8081
+  s3_compat_api_endpoint: ""
+  security_context: {}
+  service_type: ClusterIP
+  target_device: CPU
+  https_proxy: http://proxy-chain.intel.com:911
```
Comment (mzegla) [inline extras/openvino-operator-openshift/tests/sample1.yaml:32]: Should intel proxy information be displayed here?

File: extras/openvino-operator-openshift/tests/sample2.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  nireq: ""
+  plugin_config: '{\"CPU_THROUGHPUT_STREAMS\":\"CPU_THROUGHPUT_AUTO\"}'
+  replicas: 1
+  resources: {}
+  rest_port: 8081
+  s3_compat_api_endpoint: ""
+  security_context: {}
+  service_type: ClusterIP
+  target_device: CPU
+  https_proxy: http://proxy-chain.intel.com:911
```
Comment (mzegla) [inline extras/openvino-operator-openshift/tests/sample2.yaml:29]: Should intel proxy information be displayed here?

File: extras/ovms-operator/Makefile [hunk 5]
```diff
@@ -14,95 +14,176 @@
...
-VERSION ?= 0.1.0
-# Default bundle image tag
-BUNDLE_IMG ?= controller-bundle:$(VERSION)
-# Options for 'bundle-build'
+# VERSION defines the project version for the bundle.
+# Update this value when you upgrade the version of your project.
+# To re-generate a bundle for another specific version without changing the standard setup, you can:
+# - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
+# - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
+VERSION ?= 0.0.1
```
Comment (mzegla) [inline extras/ovms-operator/Makefile:22]: Before I saw version 0.1.0. Does this concern the same thing? If so perhaps we should have the same way of defining version (0.1.0 or 0.0.1). 
btw. shouldn't it be 0.0.2 with the new version?

File: extras/ovms-operator/Makefile [hunk 6]
```diff
@@ -14,95 +14,176 @@
...
+# The image tag given to the resulting catalog image (e.g. make catalog-build CATALOG_IMG=example.com/operator-catalog:v0.2.0).
+CATALOG_IMG ?= $(IMAGE_TAG_BASE)-catalog:v$(VERSION)
+
+# Set CATALOG_BASE_IMG to an existing catalog image tag to add $BUNDLE_IMGS to that image.
+ifneq ($(origin CATALOG_BASE_IMG), undefined)
+FROM_INDEX_OPT := --from-index $(CATALOG_BASE_IMG)
+endif
+
+# Build a catalog image by adding bundle images to an empty catalog using the operator package manager tool, 'opm'.
+# This recipe invokes 'opm' in 'semver' bundle add mode. For more information on add modes, see:
```
Comment (mzegla) [inline extras/ovms-operator/Makefile:180]: semver?

**Review by rasapala — CHANGES_REQUESTED** [review]
https://github.com/openvinotoolkit/model_server/pull/721/files

**Review by rasapala — APPROVED** [review]
LGTM

**Review by mzegla — APPROVED** [review]

**Pull request merged**